src/content/docs/merchants/get-started/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Introduction
+description: Learn how to set up your merchant account and configure your storefront.
+---
+
+import CardGrid from '@components/CardGrid.astro';
+import LinkCard from '@components/LinkCard.astro';
+
+Welcome to the Merchant Documentation! This section is designed to help you set up and manage your storefront effectively. Whether you're new to the platform or looking to optimize your store, you'll find all the resources you need here.
+
+<CardGrid>
+  <LinkCard
+    noborder
+    icon="seti:video"
+    title="Multi-store setup"
```

Comment:
You're setting yourself up for needless churn by making Multi-store setup a top level category. I left comments in another PR stating that content from the  Enriching dropins and Experimenting with dropins topics needs to be moved to the Merchants section. There needs to be an overview about the starter content, what it can be configured to do, and how it ties into the boilerplate code.

Multi-store setup is just one facet of the starter content and should not be a container for terms and conditions info.